.about.yml

```diff
@@ -0,0 +1,116 @@
+---
+# .about.yml project metadata
+#
+# Short name that acts as the project identifier (required)
+name: laptop
+
+# Full proper name of the project (required)
+full_name: Laptop Script
+
+# The type of content in the repo
+# values: app, docs, policy
+type: app
+
+# Describes whether a project team, working group/guild, etc. owns the repo (required)
+# values: guild, working-group, project
+# owner_type:
+
+# Name of the main project repo if this is a sub-repo; name of the working group/guild repo if this is a working group/guild subproject
+# parent:
+
+# Maturity stage of the project (required)
+# values: discovery, alpha, beta, live, sunset, transfer, end
+stage: live
+
+# Description of the project
+description: >
+  A shell script that turns your Mac into an awesome web development
+  machine.
+
+# Tags that describe the project or aspects of the project
+tags:
+- automation
+
+# Should be 'true' if the project has a continuous build (required)
+# values: true, false
+testable: true
+
+# Team members contributing to the project (required)
+# Items:
+# - github: GitHub user name
+#   id: Internal team identifier/user name
+#   role: Team member's role; leads should be designated as 'lead'
+team:
+- github: monfresh
+  id: Moncef Belyamani
+  role: lead
+- github: afeld
+  id: Aidan Feldman
+  role: developer
```

Comment:
Ha! Well, you've been reviewing most of my PRs and you submitted a feature request, so I thought I'd include you, but I can remove you if you want.